examples/NiO/data/data_1/type.raw

```diff
@@ -0,0 +1 @@
+0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2
```

Comment:
The number of data files is too much. As it is a simple example, one set is enough.